dominode-bootstrapper/dominode_bootstrapper/geonodeadmin.py

```diff
@@ -220,36 +288,47 @@ def create_geofence_admin_rule(
 
 @app.command()
 def bootstrap(
-        base_url: str = DEFAULT_BASE_URL,
-        username: str = DEFAULT_USERNAME,
-        password: str = DEFAULT_PASSWORD
+        geonode_base_url: str = DEFAULT_GEONODE_BASE_URL,
+        geoserver_base_url: str = DEFAULT_GEOSERVER_BASE_URL,
+        geonode_username: str = DEFAULT_GEONODE_USERNAME,
+        geonode_password: str = DEFAULT_GEONODE_PASSWORD,
+        geoserver_username: str = DEFAULT_GEOSERVER_ADMIN_USERNAME,
+        geoserver_password: str = DEFAULT_GEOSERVER_ADMIN_PASSWORD
 ):
     """Perform initial bootstrap of GeoNode and GeoServer"""
     internal_group_name = 'dominode-internal'
     with httpx.Client() as client:
-        geonode_manager = GeoNodeManager(client, base_url, username, password)
+        geonode_manager = GeoNodeManager(client, geonode_base_url, geonode_username, geonode_password)
         geonode_manager.login()
         existing_groups = geonode_manager.get_existing_groups()
+        geoserver_manager = GeoServerManager(client, geoserver_base_url, geoserver_username, geoserver_password)
+
         for department in DepartmentName:
             _add_department(
                 geonode_manager,
                 department, [i['title'] for i in existing_groups]
             )
+            typer.echo(f'Bootstrapping deparment {department.value} in geoserver')
+            _bootstrap_department_in_geoserver(
+                geoserver_manager,
+                department
+            )
```

Comment:
it seems that `_add_department()` is already calling `_bootstrap_department_in_geoserver()` internally. It seems like calling it twice for the same department is a bug right?

Reply:
yes